cli/cmd/bb/bb.go

```diff
@@ -31,13 +32,32 @@ func die(exitCode int, err error) {
 	os.Exit(exitCode)
 }
 
-func runBazelAndDie(args []string) {
+func runBazelAndDie(args []string, opts *autoconfig.BazelOpts) {
 	// Now run bazel.
 	gcs := &repositories.GCSRepo{}
 	gitHub := repositories.CreateGitHubRepo(core.GetEnvOrConfig("BAZELISK_GITHUB_TOKEN"))
 	// Fetch releases, release candidates and Bazel-at-commits from GCS, forks from GitHub
 	repos := core.CreateRepositories(gcs, gcs, gitHub, gcs, gitHub, true)
 
+	if opts.EnableRemoteBazel {
+		repoConfig, err := remotebazel.Config(".")
+		if err != nil {
+			log.Fatalf("config err: %s", err)
+		}
+
+		ctx := context.Background()
```

Comment:
nit: if you let runBazelAndDie take a ctx, you can move the ctx from L101 up and pass that rather than making a new one